install/mysql/plugin_fusioninventory-empty.sql

```diff
@@ -915,6 +915,7 @@ CREATE TABLE `glpi_plugin_fusioninventory_deploygroups_dynamicdatas` (
   `plugin_fusioninventory_deploygroups_id` int(11) NOT NULL DEFAULT '0',
   `fields_array` text DEFAULT NULL,
   `can_update_group` tinyint(1) NOT NULL DEFAULT '0',
+  `computers_id_cache` text DEFAULT NULL,
```

Comment:
I think about that too, but postgresql use `bytea` instead `blob` and the `text` is unlimited in size instead MySQL, so in all cases, will need modify it :p 
I preferer `longtext` now